cli/Makefile

```diff
@@ -104,10 +113,6 @@ publish: clean prepublish build
 	cd $(CLI_DIR)/../packages/create-turbo && npm publish --tag $(TURBO_TAG)
 	cd $(CLI_DIR)/../packages/turbo-codemod && npm publish --tag $(TURBO_TAG)
 
-	cd $(CLI_DIR)/../scripts && ./await-publish.js
-	pnpm install --filter=turbo --ignore-scripts
-	git commit -anm "Update lockfile"
-
```

Comment:
I never found a method to make this work which provided actual guarantees that a second request after a successful request to the registry would return the correct information.

They appear to have an "eventually consistent" model that makes this non-viable.